### PR TITLE
chore: re-enable NCT page in navigation and baskets on portfolio page

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1497

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/.env.example [hunk 1]
```diff
@@ -11,8 +11,6 @@ REACT_APP_LEDGER_CHAIN_ID=regen-redwood-1
 REACT_APP_LEDGER_CHAIN_NAME="Regen - Redwood Testnet"
 REACT_APP_BLOCK_EXPLORER=https://redwood.regen.aneka.io
 REACT_APP_API_URI=http://localhost:5000
-# This is a feature flag to show/hide NCT basket related link in the header menu/footer.
-REACT_APP_NCT_BASKET=
```
Comment (blushi) [inline web-registry/.env.example:15]: decided to rm the feature flag completely since I don't think we'll want to disable nct again in the future...

**Comment by blushi** [comment]
@erikalogie testable from https://deploy-preview-1828--regen-registry.netlify.app/

**Comment by erikalogie** [comment]
@blushi this looks good. The only thing I'm not sure about is if this work should be merged into dev yet to be included in the release. Seems like we still need to wait for some things to come together according to this thread: https://regen-network.slack.com/archives/C0378JLQ7RN/p1679373515389199

@clevinson @ryanchristo this task makes the NCT page more visible on the app, when do we actually want that to happen?

File: web-registry/src/hooks/useQueryBaskets.ts [hunk 2]
```diff
@@ -15,7 +13,7 @@ export default function useQueryBaskets(): QueryBasketsResponse | undefined {
   const [baskets, setBaskets] = useState<QueryBasketsResponse | undefined>();
 
   useEffect(() => {
-    if (!api?.queryClient || !nctBasket) return;
+    if (!api?.queryClient) return;
 
     const queryClient: QueryClientImpl = new QueryClientImpl(api.queryClient);
```
Comment (flagrede) [inline web-registry/src/hooks/useQueryBaskets.ts:18]: Oh, we still have old fetching logic here.

Comment (blushi) [reply web-registry/src/hooks/useQueryBaskets.ts:18]: should we add a task to the tech backlog to tackle this?

Comment (flagrede) [reply web-registry/src/hooks/useQueryBaskets.ts:18]: I think we already have one to check the whole codebase for the remaining old fetching logic.
We could reference this file there.

Comment (blushi) [reply web-registry/src/hooks/useQueryBaskets.ts:18]: referenced there

**Pull request merged**